www/generate_benchmark_list.js

```diff
@@ -20,9 +20,19 @@ function loadTestListFromJSON(callback) {
 
 function generateTestListHTML(directory, tests) {
     tests.sort();
+    if (directory == "sql") {
```

Comment:
Not really because the other tests test the speed of an implementation detail more so than any action a user would take.